remember.py

```diff
@@ -0,0 +1,20 @@
+from PIL import Image
+
+
+def encrypted_message(path: str) -> str:
+    """
+    Extracts an encrypted message from an image.
+
+    Parameters:
+        path (str): The path to the image file.
+
+    Returns:
+        str: The extracted encrypted message.
+    """
+    image = Image.open(path)
+    width, height = image.size
+    return "".join(chr(y) for x in range(width) for y in range(height) if image.getpixel((x, y)) == 1)
+
+
+print(encrypted_message('C:\\Users\\yisra\\Downloads\\code.png'))
```

Comment:
Next time add a relative path and upload the image

Reply:
Done